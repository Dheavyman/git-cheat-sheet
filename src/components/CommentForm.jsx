import React from 'react';

/**
 * Comment form component
 *
 */
const CommentForm = () => (
  <div className="row">
    <div className="input-field col s12">
      <textarea id="comment" className="materialize-textarea" disabled />
      <label htmlFor="comment">Add comment</label>
    </div>
  </div>
);

export default CommentForm;
