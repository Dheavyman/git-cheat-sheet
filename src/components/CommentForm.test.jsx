import React from 'react';
import TestRenderer from 'react-test-renderer';

import CommentForm from './CommentForm';

describe('CommentForm component', () => {
  it('should render component successfully', () => {
    const renderer = TestRenderer.create(<CommentForm />);

    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
