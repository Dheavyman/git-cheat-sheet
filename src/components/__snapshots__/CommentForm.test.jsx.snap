// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentForm component should render component successfully 1`] = `
<div
  className="row"
>
  <div
    className="input-field col s12"
  >
    <textarea
      className="materialize-textarea"
      disabled={true}
      id="comment"
    />
    <label
      htmlFor="comment"
    >
      Add comment
    </label>
  </div>
</div>
`;
